src/screens/Home/Home.js

```diff
@@ -293,6 +304,22 @@ class HomeScreen extends React.Component<Props, State> {
     );
   };
 
+  renderUser = () => {
+    const { user, navigation } = this.props;
+    const userImageUri = user.profileImage ? `${user.profileImage}?t=${user.lastUpdateTime || 0}` : null;
+    return (
+      <UserButton key="user" onPress={() => { navigation.navigate(MANAGE_USERS_FLOW); }}>
```

Comment:
No need for this component here - `ProfileImage` has onClick prop itself.
Unless you'd like to increase clickable area (but now this wrapper acts just like clickable area).

And maybe there's no need to create styled component for ProfileImage? `HeaderProfileImage` is not adding any styles to the component.